docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+```bash
+pip install --pre --upgrade bigdl-spark3
+```
+
+__Note:__
+* Using Conda to install BigDL will automatically install libraries including `conda-pack`, `torch`, `torchmetrics`, `torchvision`, `pandas`, `numpy`, `pyspark==3.1.2`, and etc.
+* You can install BigDL Orca built on top of Spark 3.1.2 as follows:
+
+    ```bash
+    # Install the latest release version
+    pip install bigdl-orca-spark3
+
+    # Install the latest nightly build version
+    pip install --pre --upgrade bigdl-spark3-orca
+
+    # You need to install torch and torchvision manually
+    pip install torch torchvision
+    ```
+
+    Installing bigdl-orca-spark3 will automatically install `pyspark==3.1.2`.
+
+* You also need to install any additional python libraries that your application depends on in this Conda environment.
+
+* It's only need for you to install needed Python libraries using Conda, since the BigDL K8s container has already setup `JAVA_HOME`, `BIGDL_HOME`, `SPARK_HOME`, `SPARK_VERSION`, etc.
+
+Please see more details in [Python User Guide](https://bigdl.readthedocs.io/en/latest/doc/UserGuide/python.html).
+
+
+# 4. Prepare Dataset
+To run the example provided by this tutorial on K8s, you should upload the dataset to a network file system (NFS).
+
+Please download the Fashion-MNIST dataset manually on your __Host Node__ (where you launch docker image). 
+
+```bash
+# PyTorch official dataset download link
+git clone https://github.com/zalandoresearch/fashion-mnist.git
+
+mv /path/to/fashion-mnist/data/fashion /bigdl/nfsdata/dataset
+```
+
+# 5. Prepare Custom Modules
+Spark allows to upload Python files(`.py`), and zipped Python packages(`.zip`) to the executors by setting `--py-files` option in Spark scripts or `extra_python_lib` in `init_orca_context`. 
+
+The FasionMNIST example needs to import modules from `model.py`.
+* When using `python` command, please specify `extra_python_lib` in `init_orca_context`.
+    ```python
+    import os
+    from bigdl.orca import init_orca_context, stop_orca_context
+    from model import model_creator, optimizer_creator
+
+    conf={
+          "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName":"nfsvolumeclaim",
+          "spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path": "/bigdl/nfsdata",
+        }
+
+    init_orca_context(cluster_mode="k8s-client", num_nodes=2, cores=2, memory="2g",
+                      master=os.environ.get("RUNTIME_SPARK_MASTER"),
+                      container_image="intelanalytics/bigdl-k8s:latest",
+                      extra_python_lib="model.py", conf=conf)
+    ```
+
+    __Note:__
+    * Please upload the extra Python dependency files to NFS and use it from NFS instead (see __Section 6.1.2__).
+        ```python
+        init_orca_context(cluster_mode, num_nodes, cores, memory, master,
+                          container_image, conf, extra_python_lib="file:///bigdl/nfsdata/model.py")
+        ```
```

Comment:
This part is unclear. The first code segment uses local path while the next one uses nfs path.
Will `model.py` work?